clang-tools-extra/test/clang-tidy/checkers/modernize/use-cpp-style-comments.cpp

```diff
@@ -0,0 +1,16 @@
+// RUN: %check_clang_tidy -std=c++11 %s modernize-use-cpp-style-comments %t
+
+static auto PI = 3.14159265; /* value of pi upto 8 decimal places */
+// CHECK-MESSAGES: :[[@LINE-1]]:30: warning: use C++ style comments '//' instead of C style comments '/*...*/' [modernize-use-cpp-style-comments]
```

Comment:
If you are going to emit fixes in a check you must demonstrate those fixes in the test files using the `// CHECK-FIXES` directive